chapters/Index.md

```diff
@@ -35,24 +35,2394 @@ Jintek           <info@jintek.consulting>
+
+The type of the ::BTCR2 Beacon::  determines how these Beacon Signals SHOULD be constructed and processed to validate a set of ::BTCR2 Updates:: against the 32 bytes contained within the Beacon Signal.
+
+Beacon Signals
+
+: ::Beacon Signal::
+
+Beacon Type
+
+: The type of a ::BTCR2 Beacon::. The Beacon Type defines how ::BTCR2 Update Announcements:: are included within a ::Beacon Signal:: broadcast on the Bitcoin network. It also defines how the content committed within ::BTCR2 Update Announcements:: can be verified against the ::Beacon Signal::.
+
+Beacon Types
+
+: ::Beacon Type::
+
+BTCR2 Beacon
+
+: A abstract mechanism, identified by a ::Beacon Address::, that is included as a service in a DID document to indicate to resolvers that spends from this address, called ::Beacon Signals::, should be discovered and checked for ::BTCR2 Update Announcements::.
+
+BTCR2 Beacons
+
+: ::BTCR2 Beacon::
+
+: ::BTCR2 Beacon::
+
+BTCR2 Update
+
+: A capability invocation secured using Data Integrity that invokes an authorization capability to update a specific **did:btcr2** DID document. This capability invocation Data Integrity proof secures the ::Unsecured BTCR2 Update:: document.
+
+BTCR2 Updates
+
+: ::BTCR2 Update::
+
+BTCR2 Update Announcement
+
+: A 32 byte SHA256 hash committing to a ::BTCR2 Update:: that has been broadcast by a ::BTCR2 Beacon:: in an ::Authorized Beacon Signal::. ::Beacon Signals:: can include one or more BTCR2 Update Announcements. How ::Beacon Signals:: include announcements is defined by the ::Beacon Type::.
+
+BTCR2 Update Announcements
+
+: ::BTCR2 Update Announcement::
+
+Contemporary DID Document
+
+: The DID document that is contemporary with a Bitcoin block at a specific blockheight.
+The Contemporary DID Document changes as a resolver traverses the blockchain and applies 
+the relevant ::BTCR2 Updates:: announced by ::Authorized Beacon Signals:: it identifies in 
+specific Bitcoin blocks.
+
+Content Addressable Storage
+
+: Content Addressable Storage (CAS) is a data storage system where content is addressable using ::Content Identifiers:: (CIDs). The InterPlanetary File System (IPFS) is an example of CAS.
+
+CAS
+
+: ::Content Addressable Storage::
+
+Content Identifier
+
+: A Content Identifier (CID) is an identifier for some digital content (e.g., a file) generated from the content itself such that for any given content and CID generation algorithm there is a single, unique, collision-resistant identifier. This is typically done through some hashing function.
+
+Content Identifiers
+
+: ::Content Identifier::
+
+CID
+
+: ::Content Identifier::
+
+CIDs
+
+: ::Content Identifier::
+
+Contemporary Blockheight
+
+: The blockheight of consideration when walking the provenance of a series of DID updates. A DID document's contemporary time is the Signal Time of the ::Beacon Signal:: that announced the last ::BTCR2 Update:: applied to the DID document.
```

Comment:
```suggestion
```

My sense is we can remove this term. We dont currently reference it anywhere, and I think Contemporary DID Document captures the concept well enough